### PR TITLE
Nominate xpivarc as an approver

### PR DESCRIPTION
**Pull request by stu-gott**
Signed-off-by: Stu Gott <sgott@redhat.com>

**What this PR does / why we need it**:

Nominating @xpivarc as an approver. 

**Special notes for your reviewer**:

Community guidelines on eligibility:

https://github.com/kubevirt/community/blob/main/membership_policy.md#approver

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by jean-edouard** [comment]
I approve, I think @xpivarc largely meets the eligibility criteria. Adding a hold to give other approvers time to review.
/approve
/hold

**Comment by davidvossel** [comment]
/approve

This has my support.

**Review by dhiller — APPROVED** [review]
/approve

:tada: :rocket:

**Pull request merged**
